boards/imix/src/main.rs

```diff
@@ -327,8 +326,9 @@ pub unsafe fn reset_handler() {
          &sam4l::gpio::PC[28], // P5
          &sam4l::gpio::PC[27], // P6
          &sam4l::gpio::PC[26], // P7
-         &sam4l::gpio::PC[25], // P8
-         &sam4l::gpio::PC[25]], // Dummy Pin (regular GPIO)
+         &sam4l::gpio::PA[20], // P8
+         &sam4l::gpio::PA[20], // Dummy Pin (regular GPIO)
```

Comment:
I suspect @shaneleonard was just `sed`ing here, but good question: why do we have this dummy pin in the first place?